module/zfs/spa.c

```diff
@@ -3100,6 +3218,21 @@ spa_load_impl(spa_t *spa, uint64_t pool_guid, nvlist_t *config,
 		}
 	}
 
+	if (spa_feature_is_active(spa, SPA_FEATURE_ALLOCATION_CLASSES)) {
+		uint64_t class_stats_obj;
+
+		if (spa_dir_prop(spa, DMU_POOL_NORMAL_CLASS_STATS,
+		    &class_stats_obj) == 0) {
+			metaslab_class_enable_stats(spa->spa_normal_class,
+			    class_stats_obj);
```

Comment:
aha, so this is loading the stats, not just enabling them!  Please rename.

Reply:
renamed